### ISSUE TITLE
failing rest-backend test case test_service_convertengine

### ISSUE THREAD

**Issue by BernhardDenner** [ElektraInitiative/libelektra#1636]
# Describe what you wanted to do

Run all test cases for the rest-backend. The TC test_service_convertengine failed with error message

```
C++ exception with description "basic_string::substr: __pos (which is 32) > this->size() (which is 12)" thrown in the test body.
```

- Elektra Version: master
- Versions of other relevant software?

**Comment by markus2330** [comment]
Is this now fixed due to #1637?

**Comment by markus2330** [comment]
@BernhardDenner Is this now fixed due to #1637?

**Comment by markus2330** [comment]
I assume it is fixed now.
